pom.xml

```diff
@@ -69,24 +69,25 @@
         <!-- versions -->
 
         <io.monix.version>3.1.0</io.monix.version>
-        <kafka-clients.version>2.3.0</kafka-clients.version>
-        <embedded-kafka>2.3.0</embedded-kafka>
-        <json4s-native.version>3.6.0</json4s-native.version>
-        <json4s-jackson.version>3.6.1</json4s-jackson.version>
-        <json4s-ext>3.6.0</json4s-ext>
+        <kafka-clients.version>3.3.2</kafka-clients.version>
```

Comment:
```suggestion
        <!-- Since 0.10.0.0, Kafka has a "bidirectional" client compatibility policy.
               https://kafka.apache.org/protocol#protocol_compatibility 
        -->
        <kafka-clients.version>3.3.2</kafka-clients.version>
```